configs/mae/benchmarks/vit-huge-p14_8xb128-ds-zero3-coslr-50e_in1k.py

```diff
@@ -0,0 +1,49 @@
+_base_ = [
```

Comment:
```suggestion
_base_ = ['./vit-huge-p14_8xb128-coslr-50e_in1k.py']
```
only one file is enough, like https://github.com/open-mmlab/mmpretrain/pull/1675/files